bin/match_enc

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+require 'match'
+
+# CLI to encrypt/decrypt files using fastlane match encryption layer
+
+def usage
+  puts("USAGE: [encrypt|decrypt] input_path password [output_path]")
```

Comment:
Hmm, I wouldn't put the password in the shell history. Maybe `read -s` it?

Reply:
maybe. If I have to start making this tool a bit more complex, I am wondering if I should rewrite it completely, maybe as a fastlane action. The name was originally to be close to `openssl enc`.